mlir/test/Dialect/Bufferization/canonicalize.mlir

```diff
@@ -267,21 +279,6 @@ func.func @alloc_tensor_canonicalize() -> (tensor<4x5x?xf32>) {
 
 // -----
 
-func.func @dealloc_canonicalize_clone_removal(%arg0: memref<?xindex>) -> memref<*xf32> {
```

Comment:
Why is this test removed?

Reply:
Because this copy is not cast compatible, I indeed should not have directly deleted it. I have re-implemented this test using another approach.